etcd-client/src/kv/mod.rs

```diff
@@ -109,7 +109,7 @@ impl KvInner {
             cache,
             cache_enable,
             watch_sender: watch_req_sender,
-            cache_availability: RwLock::new(()),
+            cache_availability: AtomicBool::from(true),
```

Comment:
Any reason we change to AtomicBool here?

Reply:
RwLock is a little heavy than AtomicBool. We use function call sequence to make sure consistency.